config/protocol.mainnet.yml

```diff
@@ -25,14 +25,16 @@ ProtocolConfiguration:
 
 ApplicationConfiguration:
   DBConfiguration:
-    Type: "leveldb" #other options: 'inmemory','redis'
+    Type: "leveldb" #other options: 'inmemory','redis','boltdb'
     LevelDBOptions:
       DataDirectoryPath: "./chains/mainnet"
     RedisDBOptions:
       DataDirectoryPath: "./chains/mainnet"
       Addr: "localhost:6379"
       Password: ""
       DB: 0
+    BoltDBOptions:
+      FilePath: "./chains/mainnet/bolt"
```

Comment:
Maybe we can simplify it to `./chains/mainnet.bolt` and not mess with LevelDB dir? This way they could coexist peacefully.

Reply:
sure

Reply:
can't leave chains package since boltdb couldn't create new dir in case it's missing =/